qemu/tests/cfg/cpu_device_hotplug_maximum.cfg

```diff
@@ -0,0 +1,38 @@
+- cpu_device_hotplug_maximum:
+    virt_test_type = qemu
+    type = cpu_device_hotplug_maximum
+    no ovmf
+    no RHEL.6
+    only Linux
+    only x86_64 ppc64 ppc64le
+    required_qemu = [2.6.0, )
+    start_vm = no
+    qemu_sandbox = on
+    disallow_pcpu_overcommit = yes
+    vcpu_maxcpus = 0
+    smp = 1
+    q35:
+        machine_type_extra_params = "kernel-irqchip=split"
+        extra_params = "-device intel-iommu,intremap=on,eim=on"
+    variants:
+        - max_socket:
+            vcpu_sockets = 0
+            vcpu_cores = 1
+            vcpu_threads = 1
+        - max_core:
+            vcpu_sockets = 1
+            vcpu_cores = 0
+            vcpu_threads = 1
+        - max_thread:
+            vcpu_sockets = 1
+            vcpu_cores = 1
+            vcpu_threads = 0
+            ppc64, ppc64le:
+                smp = 8
+                vcpu_threads = 8
+                vcpu_cores = 0
+    variants:
+        - @default:
+        - with_hugepages:
+            hugepage = yes
```

Comment:
setup_hugepages = yes

Reply:
when `hugepage = yes`, `setup_hugepages` will be `yes`
https://github.com/avocado-framework/avocado-vt/blob/363a95460c11e86b9feafadeb4e15a409f6a83ca/virttest/env_process.py#L1120-L1121